workshop/exercise-1/1-questions.md

```diff
@@ -1,98 +1,101 @@
 # HTML Comprehension Questions
 
+this is a test
 
 ## Q1 - For each of the following HTML documents, is the HTML valid?
 
 Type true/false in the provided [ ].
 
-a) [ ] `<div><span>hello</div></span>`
+a) [false] `<div><span>hello</div></span>`
 
-b) [ ]
+b) [false]
 
 ```html
 <ul>
 <li>one</li>
 </ol>
 ```
 
-c) [ ] `<ul></ul><img/><ol><li>one</li></ol>`
-
+c) [false] `<ul></ul><img/><ol><li>one</li></ol>`
 
 ## Q2 - What is a screenreader and why should we care about them?
 
 _Feel free to use the powers of Google here, but please provide link(s) to your source(s)_
 
+A screen reader is a form of assistive technology [1] that is essential to people who are blind, as well as useful to people who are visually impaired, illiterate, or have a learning disability.[2] Screen readers are software applications that attempt to convey what people with normal eyesight see on a display to their users via non-visual means, like text-to-speech,[3] sound icons,[4] or a Braille device.[5]
 
-
-
-
+[1] "Types of Assistive Technology Products". Microsoft Accessibility. Retrieved June 13, 2016.
+[2] "Screen Readers and how they work with E-Learning". Virginia.gov. Archived from the original on November 13, 2018. Retrieved March 31, 2019.
+[3] "Hear text read aloud with Narrator". Microsoft. Retrieved June 13, 2016.
+[4] Coyier, Chris (October 29, 2007). "Accessibility Basics: How Does Your Page Look To A Screen Reader?". CSS-Tricks. Retrieved June 13, 2016.
+[5] "Screen Readers Technology and Refreshable Braille Displays". AFB. Retrieved June 13, 2016.
 
 ## Q3 - For each of the following cases, which tags will be needed?
 
 a) You want to create a webpage with the photos from your latest vacation
 
+img, map, area, picture
 
 b) You want to create a website that lists all the art gallery websites in your city and links to their website.
 
+a, li, ol, ul
 
 c) You want to sell designer hats. You need to receive orders from the user.
 
-
+form, input
 
 ## Q4 - Can a button be a child of a button? Explain your reasoning
 
-
-
-
-
+- technically yes.
+- should it? no.
+- a button is used most in executing an action or a command. if there are buttons that are within buttons, it will most likely cause issues.
 
 ## Q5 - What is the most generic tag you can use?
 
-
-
+- div
 
 ## Q6 - What do the following achronyms stand for?
 
-a) `a`
-
-b) `ol`
+a) `a` defines a hyperlink
```

Comment:
yes you are correct in that it defines a hyperlink. the full form is however **anchor**